frontend/src/pages/LogInPage/LogInPage.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './LogInPage.css';
```

Comment:
Note that this type of import will be applied globally (whole app).
 So make sure your classname is unique or you may want to use modular css next time